### PR TITLE
feat: chrono run knob springloaded in the 3rd position

### LINKED ISSUE #5766: Chrono RST switch should be spring loaded
### Aircraft Version

Experimental

### Build info

```json
{
  "built": "2021-08-31T04:29:40+00:00",
  "ref": "experimental",
  "sha": "5cb4212a6fa7af59e0241164a4da99091998cd44",
  "actor": "Benjozork",
  "event_name": "manual"
}
```

### Describe the bug

The switch for the chrono timer reset currently is a permanent switch position.

### Expected behavior

The switch for the chrono timer reset should be a momentary, spring loaded switch position. So when turning the switch to the RST position it should spring back to the STP position.

### Steps to reproduce

1. Turn chrono timer switch to RST.
2. switch stays at this position

### References (optional)

image

### Additional info (optional)

_No response_

### PR DESCRIPTION
**Pull request by ImenesFBW**
Fixes #5766

## Summary of Changes

Makes the CHRONO RUN KNOB springloaded in the third position and adds dynamic tooltips relative to knob position and cursor orientation. 

## Testing instructions

Play around with the chrono run knob, and chrono in general, and make sure all the functions are as before, with the added feature of the knob being momentary in the third (reset) position and tooltips being displayed depending on knob position and cursor orientation. 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml [hunk 1]
```diff
@@ -4626,6 +4625,22 @@
...
+                    <FREQUENCY>10</FREQUENCY>
+                    <UPDATE_CODE>
+                        (L:A32NX_CHRONO_ET_SWITCH_POS, number) 2 == if{
+                            (L:A32NX_CHRONO_MOMENTARY) 1 + (&gt;L:A32NX_CHRONO_MOMENTARY)
+                        }
+                            els{ 0 (&gt;L:A32NX_CHRONO_MOMENTARY)
+                            }
+                        (L:A32NX_CHRONO_MOMENTARY) 10 &gt; if{ 1 (&gt;L:A32NX_CHRONO_ET_SWITCH_POS, number)
+                        }
+                    </UPDATE_CODE>
```
Comment (tracernz) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:4643]:
```suggestion
                    <UPDATE_CODE type="rnp">
                        alias etSwitchPos = (L:A32NX_CHRONO_ET_SWITCH_POS, number);
                        alias momentaryTimer = (O:A32NX_CHRONO_MOMENTARY, number);

                        if etSwitchPos == 2 {
                            momentaryTimer += 1;

                            if momentaryTimer > 10 {
                                etSwitchPos = 1;
                            }
                        } else if momentaryTimer > 0 {
                          momentaryTimer = 0;
                        }
                    </UPDATE_CODE>
```

Comment (tracernz) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:4643]:
```suggestion
                    <UPDATE_CODE type="rnp">
                        alias etSwitchPos = (L:A32NX_CHRONO_ET_SWITCH_POS, number);
                        alias momentaryTimer = (O:A32NX_CHRONO_MOMENTARY, number);

                        if etSwitchPos == 2 {
                            momentaryTimer += 1;

                            if momentaryTimer > 10 {
                                etSwitchPos = 1;
                            }
                        } else if momentaryTimer > 0 {
                          momentaryTimer = 0;
                        }
                    </UPDATE_CODE>
```

Comment (tracernz) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:4643]:
```suggestion
                    <UPDATE_CODE type="rnp">
                        alias etSwitchPos = (L:A32NX_CHRONO_ET_SWITCH_POS, number);
                        alias momentaryTimer = (O:A32NX_CHRONO_MOMENTARY, number);

                        if etSwitchPos == 2 {
                            momentaryTimer += 1;

                            if momentaryTimer > 10 {
                                etSwitchPos = 1;
                            }
                        } else if momentaryTimer > 0 {
                          momentaryTimer = 0;
                        }
                    </UPDATE_CODE>
```

Comment (tracernz) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml:4643]:
```suggestion
                    <UPDATE_CODE type="rnp">
                        alias etSwitchPos = (L:A32NX_CHRONO_ET_SWITCH_POS, number);
                        alias momentaryTimer = (O:A32NX_CHRONO_MOMENTARY, number);

                        if etSwitchPos == 2 {
                            momentaryTimer += 1;

                            if momentaryTimer > 10 {
                                etSwitchPos = 1;
                            }
                        } else if momentaryTimer > 0 {
                          momentaryTimer = 0;
                        }
                    </UPDATE_CODE>
```

**Review by tracernz — APPROVED** [review]

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #6661
Tier of Testing  : 1
Date             : 09/02/2022

Testing Process: 
Run the chronometer to check that it functions correctly.
Stopped, reset and back to run.
Checked for correct tooltips appearing.

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
LGTM 🚀 

Media:       
image

image

**Pull request merged**